apps/faucet/src/App/Faucet.tsx

```diff
@@ -159,6 +161,7 @@ export const FaucetForm: React.FC<Props> = ({ isTestnetLive }) => {
 
       <InputContainer>
         <Input
+          placeholder="From 1 to 1000"
```

Comment:
The faucet limit is set at build time and is not necessarily 1000, so this should use the variable `faucetLimit` instead of 1000.